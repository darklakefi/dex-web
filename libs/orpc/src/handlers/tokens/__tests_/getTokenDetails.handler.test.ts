import { getTokenDetailsHandler } from "../getTokenDetails.handler";

describe("getTokensHandler", () => {
  it.skip("should return a token details", async () => {
    const result = await getTokenDetailsHandler({
      address: "So11111111111111111111111111111111111111112",
    });

    expect(result).toBeDefined();
    expect(result).toMatchSnapshot();
  });
});
