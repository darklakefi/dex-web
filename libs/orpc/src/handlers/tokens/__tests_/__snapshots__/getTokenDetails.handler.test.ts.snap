// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`getTokensHandler > should return a token details 1`] = `
{
  "address": "So11111111111111111111111111111111111111112",
  "decimals": 9,
  "imageUrl": "https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/So11111111111111111111111111111111111111112/logo.png",
  "name": "Wrapped SOL",
  "symbol": "SOL",
  "value": "So11111111111111111111111111111111111111112",
}
`;
